app/controllers/hosts_controller.rb

```diff
@@ -684,7 +684,7 @@ def statuses
     render :json => statuses
   end
 
-  def hosts_path(*args)
+  def current_hosts_path(*args)
     if Setting[:new_hosts_page]
       new_hosts_index_page_path(*args)
     else
```

Comment:
No `super` for `current_hosts_path`, right?

Reply:
yes no super

Reply:
You had `super` on the next line but I couldn't comment it.